lib/l10n/app_en.arb

```diff
@@ -1306,5 +1306,6 @@
   "lockDownload": "Keep on Device",
   "@lockDownload": {
     "description": "Text shown on button to require an item which is already downloaded by a parent collection, preventing it from being deleted if the parent is removed."
-  }
+  },
+  "showArtistChipImage": "Show image on artist chip"
```

Comment:
Would prefer a less technical description, like "Show artist images with artist names", what do you think?

Reply:
Sure.  Maybe it should mention that it's specifically the chips and not in the main list, but I can't think of a great way to phrase that.  'with artist names' doesn't feel as specific.

Reply:
Yeah, I also struggled with that. Maybe adding a subtitle is a good idea then?

Reply:
Added subtitle: "This affects small artist image previews, such as on the player screen."